apps/condo/domains/property/components/PropertyForm/UpdatePropertyForm.tsx

```diff
@@ -7,27 +7,34 @@ import { useOrganization } from '@open-condo/next/organization'
 
 import { Button } from '@condo/domains/common/components/Button'
 import { Loader } from '@condo/domains/common/components/Loader'
+import { isSafeUrl } from '@condo/domains/common/utils/url.utils'
```

Comment:
It seems that you can redirect the user to any site. This should be fixed (it can be done in another task, but please don't forget about it)